pkg/federate/kubefed/resource_test.go

```diff
@@ -223,12 +223,12 @@ func testDistribute() {
 
 		When("on update the Kube API returns an error that is not a NotFound", func() {
 			BeforeEach(func() {
-				clientPatcher = &fakeClientWithUpdateError{fake.NewFakeClient(), "test error"}
+				clientPatcher = &fakeClientWithUpdateError{fake.NewFakeClientWithScheme(scheme, fedResource), "fake error"}
```

Comment:
Why is this change needed?

Reply:
Because in order to test update there needs to be an existing resource.